lib/batches/batch_spec.go

```
@@ -177,6 +188,11 @@ func parseBatchSpec(schema string, data []byte) (*BatchSpec, error) {
 		}
 	}
 
+	// For backward compatibility, we allow the version field to be omitted.
+	if spec.Version != "" && !slices.Contains(SupportedBatchSpecVersions, spec.Version) {
```

you will have to update src-cli with the latest version of sg/sg/lib and probably add some logic to the query runner there, too so v2 works in CLI execution mode as well

---

@eseliger Is the src-cli change a blocker for this PR? Or can we follow-up with the src-cli change even after the release?

---

We usually release those in lockstep, but I don't think it has to be in for 5.5.0, we can cut a 5.5.1 as well. We should just make sure to not let that sit for too long, as batch specs shown in the web will no longer work locally (doesn't understand `version` and silently ignores it)